x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.ts

```diff
@@ -46,12 +51,12 @@ export const getActionList = async ({
   esClient: ElasticsearchClient;
   logger: Logger;
 }): Promise<ActionListApiResponse> => {
-  const size = pageSize ?? 10;
-  const page = (_page ?? 1) - 1;
+  const size = pageSize ?? ENDPOINT_DEFAULT_PAGE_SIZE;
+  const page = _page ?? ENDPOINT_DEFAULT_PAGE;
```

Comment:
This does not look correct.  if `_page` is defined, it is `1` based, so you have to subtract 1 from it so that you convert it to a "start index number". 

`ENDPOINT_DEFAULT_PAGE` is really an Elasticsearch `from` number, which is zero based.

Reply:
Okay, so we're going with 1 based paging! I'll update it on the server and front end 😆